app/views/rooms/show.html.erb

```diff
@@ -103,8 +126,13 @@
     <p>No Images Found</p>
 <% end%>
 </div>
-<hr width="1200" align="center">
-<img src="..." class="img-fluid" alt="Map image">
+<hr width="1200" >
+<div class="mt-4 offset-2">
+    <h1>Location</h1>
+    
+    <div id="map" style="min-with:500px; "></div>
```

Comment:
update this code to this one:
`<div id="map" style="min-width:800px; max-width:1300px; height: 500px; "></div>`